frontend/src/index.ts

```diff
@@ -3,15 +3,16 @@ import { getDrivers, wrapMain } from './drivers';
 import { Component } from './interfaces';
 import { App } from './components/app';
 import { initMap } from './components/map';
-import { init3d, animate } from './components/3d';
+import { init3d, animate, registerMouseEvent } from './components/3d';
 
 const main: Component<any> = wrapMain(App);
 
 run(main as any, getDrivers());
 
-// Init map once document is loaded
+// Init map and building 3D model once document is loaded
 window.addEventListener('load', () => {
-    console.log('map loaded');
+    console.log('loading map and 3D model');
     initMap();
     init3d();
+    registerMouseEvent();
```

Comment:
I'd consider here naming: It is for app to know that the events are "mouse events" and I'm not sure if startup should  "know" at all that some events are registered. Why this is not as a part of intit3d()?

Reply:
Good point. This was there initially when the click support did not work some reason and I was trying to eliminate some loading issues.

Reply:
Removed from index.ts